index.html

```diff
@@ -8,6 +8,44 @@
     <title>Daniel Selase | Portfolio</title>
 </head>
 <body>
-    
+    <div class="container">
+        <!-- Navigation -->
+        <nav class="nav">
+            <div class="nav__logo">
+                <p>Welcome</p>
+            </div>
+            <div class="nav__links">
+                <div class="nav__links-items">
+                    <a href="#">Portfolio</a>
+                    <a href="#">About</a>
+                    <a href="# ">Contact</a>
+                </div>
+                <div class="nav__hamburger">
+                    <img src="./images/Enabled@2x.svg" alt="">
+                </div>
+            </div>
+        </nav>
+    </div>
+    <!-- Header -->
+    <header class="header">
+        <div class="container">
+            <d iv class="header__title">
+                <h1><span class="header__title-dark">Hey there. I'm Daniel Selase</span><span class="header__title-green">I'm a software developer</span></h1>
+            </d>
+            <div class="header__content">
+                <p>I can help you build a product feature or website. Look through some of my work and experience! If you like what you see and have a product you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="header__social">
+                <img src="./images/twitter.svg" alt="">
+                <img src="./images/Linkedin icon.svg" alt="">
+                <img src="./images/medium.svg" alt="">
+                <img src="./images/github.svg" alt="">
+                <img src="./images/angellist.svg " alt="">
+            </div>
+            <div class="header_arrow-down">
+                <img src="./images/Disabled.png" alt="Navigation down arrow">
+            </div>
+        </div>
+    </header>
```

Comment:
as I mentioned in the other comment, the `<header>` tag is used as a page header, it contains the site's title or logo, and since the `<section>` tag represents a generic standalone section of the document, we should use it, so kindly : 
- [ ] replace your `<header class="header">` with `<section class="header">` 🙏 
- [ ] and make sure that the content of this section is aligned with flexbox